scripts/bench.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+import argparse
+import os
+import sys
+
+from monotonic import monotonic
+
+from detect_secrets.core.usage import PluginOptions
+
+if os.name == 'posix' and sys.version_info[0] < 3:
+    import subprocess32 as subprocess
+else:
+    import subprocess
+
+
+def getArguments():
+    parser = argparse.ArgumentParser(description='Run some benchmarks.')
+    parser.add_argument(
+        'rest_args',
+        nargs=argparse.REMAINDER,
+        help='filenames to check or detect-secrets compatible arguments',
+    )
+    args = parser.parse_args()
+    return ['../.'] if len(args.rest_args) == 0 else args.rest_args
+
+
+def runBench(flags, arguments):
+    startTime = monotonic()
+    subprocess.call('detect-secrets scan'.split() + arguments + flags, stdout=subprocess.DEVNULL)
+    return monotonic() - startTime
+
+
+def printLine(name, time):
+    print('{:<20s}{:>20s}s'.format(name, str(time)))
+
+
+flags = list()
+aditionalArguments = getArguments()
+
+# Fill flags list with PluginOptions flag text
+for c, value in enumerate(PluginOptions.all_plugins, 0):
```

Comment:
Don't need to specify the `0`, because that's by default.

Reply:
Fixed!